extensions/notebook/src/jupyter/jupyterServerInstallation.ts

```diff
@@ -133,20 +133,28 @@ export class JupyterServerInstallation implements IJupyterServerInstallation {
 	private readonly _requiredPackagesSet: Set<string>;
 
 	private readonly _runningOnSAW: boolean;
+	private readonly _tsgops: boolean;
 
 	constructor(extensionPath: string, outputChannel: vscode.OutputChannel) {
 		this.extensionPath = extensionPath;
 		this.outputChannel = outputChannel;
 
 		this._runningOnSAW = vscode.env.appName.toLowerCase().indexOf('saw') > 0;
+		this._tsgops = vscode.env.appName.toLowerCase().indexOf('tsgops') > 0;
 		void vscode.commands.executeCommand(constants.BuiltInCommands.SetContext, 'notebook:runningOnSAW', this._runningOnSAW);
 
 		if (this._runningOnSAW) {
 			this._pythonInstallationPath = `${vscode.env.appRoot}\\ads-python`;
 			this._usingExistingPython = true;
 		} else {
-			this._pythonInstallationPath = JupyterServerInstallation.getPythonInstallPath();
-			this._usingExistingPython = JupyterServerInstallation.getExistingPythonSetting();
+			if (this._tsgops) {
```

Comment:
Just have this be an `else if` and `else` on the original if/else. Don't need to nest it